src/components/material/ReviewMetadata.tsx

```diff
@@ -15,14 +16,29 @@ export const usDateStringToDateObj = (date: string): Date | null => {
   return dayjs(date, "YYYY-MM-DD").toDate();
 };
 
-const ReviewMetadata: React.FC<ReviewMetadataProps> = ({ author, date }) => {
-  return (
-    <div className="review__meta mb-8">
-      {`${author || ""}${author && date ? ", " : ""}${
-        date ? dayjs(date).format("DD.MM.YYYY") : ""
-      }`}
-    </div>
-  );
+const ReviewMetadata: React.FC<ReviewMetadataProps> = ({
+  author,
+  date,
+  link
+}) => {
+  const metaDataText = (
+    returnAuthor: string | null | undefined,
```

Comment:
This is nitty gritty, but why the "return" prefix on the parameters? Are they necesarry?
Also, you `metaDataText` is actually a functional component. So you could also move that out to a separate `<MetaDataText />` component.

Reply:
The "return" prefix is there because there cannot be duplicates in one file, and we already have both Author and Date. And It is not a component because it returns just a string. Even if we put it into its own component and return that string in a React fragment, Eslint and typescript will complain. I am adamant in keeping it as is in this case.

Reply:
Since it is parameters in a function they have their own scope and won't conflict with the variables outside of it.
Weird that eslint/typescript complains about a string in a fragment. What kind of error does it throw?

Reply:
![image](https://user-images.githubusercontent.com/28546954/183837669-a956cd23-2a6f-4ef9-af1e-9d7dfef4bd80.png)